mgmtsystem_security_event/views/category_assets.xml

```diff
@@ -0,0 +1,39 @@
+<openerp>
```

Comment:
Please use the full model name for the file `mgmtsystem_security_assets_category.xml`